### ISSUE TITLE
Training script

### ISSUE THREAD

**Issue by 0xffee00** [HKUST-Aerial-Robotics/MVDepthNet#6]
Hi, Thanks for the great paper and code. 
Do you plan on releasing a training script including the geometric augmentation? 

Thanks

**Comment by WANG-KX** [comment]
Dear,

Thanks for your interest in the project.
The training is very straightforward. Releasing the training code means the training/test data will be released too. However, it is not yet possible due to the large size of the data. Do you have any specific questions about the training? I think the paper includes the learning rate, batch size, iteration numbers, and loss definition.

Regards,
Kaixuan

**Comment by 0xffee00** [comment]
Thanks for the reply. 
No specific questions yet. I have downloaded the training data, but I am somewhat new to neural networks so it might take some time to recreate the training procedure. I will post any questions that come up. Thanks.
